Update iOS partner integration framework docs (#22)
yat-labs/yat-docs: yat-docs/docs/guides/8_ios_partner_integration_framework.md

use_frameworks!

target 'MyApp' do
    pod 'YatLib', '~> 0.2.2'
end
```

## Setup

1. Create your app, add `YatLib` as a pod dependency as described above, and do a `pod install`.

> 2. `YatLib` uses deep links to return from the Yat web application back to the application. URL scheme of the deep link is agreed upon in between the Yat development team and the integration partner. Setup your deep links in your project accordingly.

Reviewer: ```suggestion
2. `YatLib` uses deep links to return from the Yat web application back to the application. The URL scheme of the deep link is agreed upon between the Yat development team and the integration partner. Setup your deep links in your project accordingly.
```

    import UIKit
    import YatLib

    final class SceneDelegate: UIResponder, UIWindowSceneDelegate {

        func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
            Yat.configuration = YatConfiguration(appReturnLink: "{app_return_link}", organizationName: "{organization_name}", organizationKey: "{organization_key}")
	    }

        func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
            guard let url = URLContexts.first?.url else { return }
            Yat.integration.handle(deeplink: url)
        }

        // ... rest of the implementation ...

    }
	```

>     App return link, organization name, and organization key will be delivered to you by the Yat development team.

Reviewer: ```suggestion
    The app return link, organization name, and organization key will be delivered to you by the Yat development team.
```

    # Usage

    `Yat` is an integration entry point. It contains all tools necessary to configure, style, integrate, and interact with API.

    ## Configuration

>     To configure the integration, you need to pass a new configuration to the `Yat.configuration` (please check the Setup section above for more information). If 

Reviewer: Trailing sentence

Reviewer: ```suggestion
    To configure the integration, you need to pass a new configuration to the `Yat.configuration` (please check the Setup section above for more information).
```

    ## Style

    You can change the style (colors, fonts, etc.) of the UI elements used by the framework by changing values stored in `Yat.style`.

    ## Integration

>     `Yat.integration` keeps all convenience methods used to show a unified UI that allows the user to connect his crypto wallet address. 

Reviewer: ```suggestion
    `Yat.integration` exposes convenience methods to present a unified UI that allows the user to connect his crypto wallet address to Yat. 
```

    To show a simple onboarding overlay, you need to:
    ```swift
	    Yat.integration.showOnboarding(onViewController: hostViewController, records: records)
    ```

>     Where `hostViewController` is a `UIViewController` which will host for modal overlay, and `records` is an array of `YatRecordInput` structures that will be attached to the user's Yat.

Reviewer: ```suggestion
    Where `hostViewController` is a `UIViewController` which will host the modal overlay, and `records` is an array of `YatRecordInput` structures that will be attached to the user's Yat.
```

>     To properly handle the response after the success. you need to add a:

Reviewer: ```suggestion
    To properly handle the response after the success. you simply add:
```

    ```swift
	    Yat.integration.handle(deeplink: url)
    ```
    in `SceneDelagate.swift`. Please check the Setup section above for more information.

    ## API

>     `Yat.api` keeps all convenience methods used to directly communicate with Yat's API. Currently, YatLib provides methods that handle API calls listed below:

Reviewer: ```suggestion
    `Yat.api` provides all the convenience methods used to directly communicate with the Yat API. Currently, YatLib provides methods that handle API calls listed below:
```
